engo_js.go

```diff
@@ -1,165 +1,127 @@
-		rect := canvas.Call("getBoundingClientRect")
-		x := float32((ev.Get("clientX").Int() - rect.Get("left").Int()))
-		y := float32((ev.Get("clientY").Int() - rect.Get("top").Int()))
-		responder.Mouse(x, y, PRESS)
-	}, false)
-
-	canvas.Call("addEventListener", "mouseup", func(ev js.Object) {
-		rect := canvas.Call("getBoundingClientRect")
-		x := float32((ev.Get("clientX").Int() - rect.Get("left").Int()))
-		y := float32((ev.Get("clientY").Int() - rect.Get("top").Int()))
-		responder.Mouse(x, y, RELEASE)
-	}, false)
-
-	canvas.Call("addEventListener", "touchstart", func(ev js.Object) {
-		rect := canvas.Call("getBoundingClientRect")
-		for i := 0; i < ev.Get("changedTouches").Get("length").Int(); i++ {
-			touch := ev.Get("changedTouches").Index(i)
-			x := float32((touch.Get("clientX").Int() - rect.Get("left").Int()))
-			y := float32((touch.Get("clientY").Int() - rect.Get("top").Int()))
-			responder.Mouse(x, y, PRESS)
-		}
-	}, false)
-
-	canvas.Call("addEventListener", "touchcancel", func(ev js.Object) {
-		rect := canvas.Call("getBoundingClientRect")
-		for i := 0; i < ev.Get("changedTouches").Get("length").Int(); i++ {
-			touch := ev.Get("changedTouches").Index(i)
-			x := float32((touch.Get("clientX").Int() - rect.Get("left").Int()))
-			y := float32((touch.Get("clientY").Int() - rect.Get("top").Int()))
-			responder.Mouse(x, y, RELEASE)
-		}
-	}, false)
-
-	canvas.Call("addEventListener", "touchend", func(ev js.Object) {
-		rect := canvas.Call("getBoundingClientRect")
-		for i := 0; i < ev.Get("changedTouches").Get("length").Int(); i++ {
-			touch := ev.Get("changedTouches").Index(i)
-			x := float32((touch.Get("clientX").Int() - rect.Get("left").Int()))
-			y := float32((touch.Get("clientY").Int() - rect.Get("top").Int()))
-			responder.Mouse(x, y, PRESS)
-		}
-	}, false)
-
-	canvas.Call("addEventListener", "touchmove", func(ev js.Object) {
-		rect := canvas.Call("getBoundingClientRect")
-		for i := 0; i < ev.Get("changedTouches").Get("length").Int(); i++ {
-			touch := ev.Get("changedTouches").Index(i)
-			x := float32((touch.Get("clientX").Int() - rect.Get("left").Int()))
-			y := float32((touch.Get("clientY").Int() - rect.Get("top").Int()))
-			responder.Mouse(x, y, MOVE)
-		}
-	}, false)
-
-	js.Global.Call("addEventListener", "keypress", func(ev js.Object) {
-		responder.Type(rune(ev.Get("charCode").Int()))
-	}, false)
-
-	js.Global.Call("addEventListener", "keydown", func(ev js.Object) {
-		key := Key(ev.Get("keyCode").Int())
-		keyStates[key] = true
-	}, false)
+	Files = NewLoader()
+	WorldBounds.Max = Point{Width(), Height()}
+}
 
-	js.Global.Call("addEventListener", "keyup", func(ev js.Object) {
-		key := Key(ev.Get("keyCode").Int())
-		keyStates[key] = false
-		// responder.Key(Key(ev.Get("keyCode").Int()), 0, RELEASE)
-	}, false)
+func SetBackground(c color.Color) {
+	if !headless {
+		r, g, b, a := c.RGBA()
+		Gl.ClearColor(float32(r), float32(g), float32(b), float32(a))
+		Gl.Clear(Gl.COLOR_BUFFER_BIT)
+	}
+}
```

Comment:
This isn't OS-specific either? If the `Gl.Clear` is required, we should also do it at the desktop-variant. 

I'd suggest moving this back to `engo.go`, because it's not OS-specific.